### DISCUSSION TITLE
Images in grid not respecting height settings (except on Firefox)

### DISCUSSION THREAD

**Discussion by versipellis** [tailwindlabs/tailwindcss#6241] [Help]
**What version of Tailwind CSS are you using?**

For example: v2.2.19

**What build tool (or framework if it abstracts the build tool) are you using?**

For example: postcss-cli 8.3.1, Next.js 10.0.9, webpack 5.28.0

**What version of Node.js are you using?**

14.17.6

**What browser are you using?**

Works fine on Firefox, bugged on Chrome, Edge, Safari, mobile

**What operating system are you using?**

Windows

**Reproduction URL**

https://play.tailwindcss.com/3mgzQNCkwK
**Describe your issue**

Setting a div to a certain height (e.g. with `h-96`) will not resize the images on the grid within to the div's maximum height (even with setting `object-fit: contain;` in the div's `style`). It works fine on Firefox, but is bugged on Chrome, Edge, Safari, and basically every other browser everyone else that's not me appears to be using :p

**Comment by adamwathan** [comment]
Hey! This is just a CSS thing, not a bug in Tailwind, as demonstrated here:

https://jsfiddle.net/y1povwu6/

Weird one for sure, doesn't make sense to me that the images wouldn't respect the grid height 🤔 If someone can figure out how to do it in vanilla CSS it should be possible in Tailwind though.

**Reply by versipellis** [reply]
Ah, very odd - thank you very much! Any ideas how to work around that?

**Reply by versipellis** [reply]
Interestingly enough, in that JSFiddle example, it actually breaks on Firefox for me now too!
